packages/wrangler/src/__tests__/secret.test.ts

```diff
@@ -95,6 +96,41 @@ describe("wrangler secret", () => {
 				expect(std.err).toMatchInlineSnapshot(`""`);
 			});
 
+			it("should create a secret", async () => {
+				writeFileSync(
+					"secret.json",
+					JSON.stringify({
+						"secret-name-1": "secret_text",
+						"secret-name-2": "secret_text",
+					})
+				);
+
+				// User counter to pass different secrets to the request mock
+				const counter = 0;
+				setMockResponse(
+					`/accounts/:accountId/workers/scripts/:scriptName/secrets`,
+					"PUT",
+					([_url, accountId]) => {
+						expect(accountId).toEqual("some-account-id");
+						counter + 1;
+
+						return { name: `secret-name-${counter}`, type: "secret_text" };
+					}
+				);
+
+				await runWrangler(
+					"secret put the-key --name script-name -j ./secret.json"
+				);
```

Comment:
What is the significance of `the-key` here? From what I can tell, it doesn't do anything, in which case we should make it so we don't pass in anything here when we have a json file as otherwise it is confusing

Reply:
Yeah I wasn't sure how to handle that since it is a positional requirement for secrets, I considered making a new bulk command altogether.

Reply:
are you allowed to have a key in the form of `something.somethingelse`? if not, we could detect from the positional arg whether we end in `.json` and then do the upload there, rather than having a separate flag?

Reply:
No. I decided to just move this into its own command.